### PR TITLE
Able to fetch todos

### PR DESCRIPTION
**Pull request by Abrahamkivosh**
Able to pull todos

### CODE REVIEW

**Comment by Abrahamkivosh** [comment]
Great I have checked the code

**Review by abrahamkivondo — APPROVED** [review]
Cool Review Approved
Clean code

File: index.js [hunk 1]
```diff
@@ -1,2 +1,4 @@
 // Added our server initialization file
-require('./server');
\ No newline at end of file
+require('./server');
+// Added our routes file
+require('./routes');
```
Comment (Abrahamkivosh) [inline index.js:4]: The previous spacing is removed successfully

Comment (Abrahamkivosh) [reply index.js:4]: New coding

**Pull request merged**